### DISCUSSION TITLE
Do I need Visual Studio to install FileMeta?

### DISCUSSION THREAD

**Discussion by hzmonte** [Dijji/FileMeta#126] [Q&A]
"Two Visual Studio solutions are provided: FileMetadata.sln and TestDriver.sln."  - does that mean in order to install this software, one needs Visual Studio ?

**Comment by Benedikt81** [comment]
No, you just need to install the .msi-package from the release: https://github.com/Dijji/FileMeta/releases (open the "Assets"-menu and download it)
